modules/ROOT/pages/index.adoc

```diff
@@ -146,55 +146,106 @@ Refer to xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypo
-
-* API Manager API v1.0 to manage an API.
-* API Manager proxy API v1.0 to manage the deployment of API proxies through API Manager.
-| Runtime Manager |	Improved error description on alert creation.
+| Component | Feature Description | Feature Version
+| Exchange 
+
+a|
+
+* Store all integration assets in one place in Anypoint Exchange, such as best practices, integration patterns, API fragments, API specifications, examples, templates, and connectors.
+* Enrich portal content using the visual editor and markdown editor.
+* Upload Open API specifications (Swagger) in Anypoint Exchange, which automatically converts to RAML for use across Anypoint Platform.
+* Upload WSDLs (SOAP APIs) in Anypoint Exchange.
+* Consume and reuse from Anypoint Studio and Anypoint Design Center.
+* Share an asset with users from other business groups.
+* Autogenerate a Mule 4.0 Design Center connector (using REST Connect) for any valid API specification for use within Anypoint Design Center.
+* View a list of dependencies (API fragments) for any API specification.
+* Version any asset published to Anypoint Exchange.
+* View dependency snippet for connectors for use in Maven, Gradle, SBT, and Ivy.
+* Publish examples and templates using Anypoint Studio 6.3 and later.
+* API portals are now part of Anypoint Exchange.
+* Create public portals for any APIs in Anypoint Exchange.
+* Automatically generate rich documentation for RAML and OAS specifications.
+* Perform interactive API use-case validation through API Notebook.
+* Use a new mocking service for testing API calls without API implementation.
+* Register clients and request access for APIs managed by Anypoint Platform.
+* Automatically index API instances and endpoints via Anypoint Exchange.
+* Share APIs externally using Anypoint Exchange portal.
+* Customize and brand features of Anypoint Exchange portal.
+* Leverage HTML support for content inserted in a markdown editor (limitations apply).
+* Save a search for organizations that provides consistent search navigation to all business group users.
+* Use personalized saved searches to save frequently used searches.
+* Copy images into the Anypoint Exchange visual editor, which eliminates the need to serve images from outside Exchange.
+* Managed categories allow Anypoint Exchange administrators and organization owners to create categories and group Exchange assets.
+* Anypoint Exchange contributors can now organize assets into different groups to improve asset browsing and discovery.
+* Custom fields allow Anypoint Exchange administrators and contributors to extend existing asset attributes through the API.
+* Reorder pages on the asset detail page.
+
+a| 
+
+* API Metadata Service v0.7.x
+* Asset Manager Service v0.8.x
+* Asset Portal Service v0.6.x
+* Assets Reviews Service v0.6.x
+* Custom Assets Facade Service v0.8.x
+* Customization Service v0.5.x
+* Dependency Resolver Service v0.6.x
+* File Service -
+* File Upload Service -
```

Comment:
Why do these two end with just a hyphen?

Reply:
Oh good catch, I need to remove those